tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,29 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
```

Comment:
Can we strip the spaces before and after these docstrings to make things more according to python-styleguides?